src/MessageBus/Handling/Implementation/HandlerProviderFromStaticArray.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types = 1);
+
+namespace Damejidlo\MessageBus\Handling\Implementation;
+
+use Damejidlo\MessageBus\Handling\HandlerType;
+use Damejidlo\MessageBus\Handling\IHandlerProvider;
+use Damejidlo\MessageBus\IMessageHandler;
+
+
+
+final class HandlerProviderFromStaticArray implements IHandlerProvider
```

Comment:
Not sure about the name... maybe `ArrayMapHandlerProvider`?

Reply:
Me neither. `ArrayMapHandlerProvider` seems better.

Reply:
Renamed both `HandlerProviderFromStaticArray` and `HandlerTypesResolverFromStaticArray`.